src/app/containers/ATIAnalytics/atiUrl.js

```diff
@@ -12,7 +12,7 @@ import {
   sanitise,
 } from '../../lib/analyticsUtils';
 
-export const atiBaseUrl = 'https://a1.api.bbc.co.uk/hit.xiti?';
+export const atiBaseUrl = () => process.env.SIMORGH_ATI_BASE_URL;
```

Comment:
Does this need to be a function? Cant you just call `process.env.SIMORGH_ATI_BASE_URL` where its used? Also then it doesnt need isolated tests. You can stub it in tests by just setting `process.env.SIMORGH_ATI_BASE_URL` in the tests